docs/BREAKINGCHANGES.md

```diff
@@ -6,151 +8,125 @@ TODO
 TODO
 
 
-### Add error on Legacy Credential over non-HTTPS for Web Cmdlets [5402](https://github.com/PowerShell/PowerShell/issues/5402)
-TODO
-
-
 ### Change $OutputEncoding to use iso-8859-1 encoding rather than ASCII [5361](https://github.com/PowerShell/PowerShell/issues/5361)
-TODO
-
+This is marked as a breaking change because the output of the pipeline is not altered as it was previously, anyone relying on ascii encoding (especially for extended ascii sequences) will no longer have that behavior.
 
-### Fix Single Value JSON null in Invoke-RestMethod [5338](https://github.com/PowerShell/PowerShell/issues/5338)
-TODO
+TODO: This is closed in favor of a new PR which simply changes the output encoding to utf8nobom, the new PR is not referenced.  Need to find and see if it is still a breaking change.
 
 
 ### Invoke-RestMethod doesn't return useful info when no data is returned. [5320](https://github.com/PowerShell/PowerShell/issues/5320)
-TODO
+When an API returns just `null`, Invoke-RestMethod was serializing this as the string `"null"` instead of `$null`.  This change fixes the logic in `Invoke-RestMethod` to properly serialize a valid single value JSON `null` literal as `$null`.
 
 
 ### Remove DCOM support from *-Computer cmdlets (replacement for 5152) [5277](https://github.com/PowerShell/PowerShell/issues/5277)
-TODO
+DCOM is not supported in corefx, therefore the *-Computer cmdlets were removed.
 
 
 ### Remove AllScope from most default aliases [5268](https://github.com/PowerShell/PowerShell/issues/5268)
-TODO
-
-
-### Fix performance issues in Add-Type [5243](https://github.com/PowerShell/PowerShell/issues/5243)
-TODO
+To speed up spope creation, AllScope was removed from most default aliases.  AllScope was left for a few frequently used aliases where the lookup was faster.
```

Comment:
Should we have different sections for impactful `Breaking Changes` and technical `Breaking Changes`?  This one falls under the latter where it shouldn't affect anyone.